kubernetes-guides/advanced-guides/kubeprism.mdx

```diff
@@ -39,7 +39,7 @@ Talos spins up a TCP loadbalancer on every machine on the `localhost` on the spe
 
 * the external cluster endpoint as specified in the machine configuration
 * for controlplane machines: `https://localhost:<api-server-local-port>` (`http://localhost:6443` in the default configuration)
-* `https://<controlplane-address>:<api-server-port>` for every controlplane machine (based on the information from [Cluster Discovery](../configure-your-talos-cluster/system-configuration/discovery.mdx))
+* `https://<controlplane-address>:<api-server-port>` for every controlplane machine (based on the information from [Cluster Discovery](../../talos/v1.10/configure-your-talos-cluster/system-configuration/discovery.mdx))
```

Comment:
Can you make these links use the latest talos release version variable? Will be easier to maintain than trying to hard code versions each time we have a release.